src/main/java/com/LetMeDoWith/LetMeDoWith/common/exception/GlobalExceptionHandler.java

```diff
@@ -19,6 +20,16 @@ protected ResponseEntity<FailResponseDto> handleRestApiException(RestApiExceptio
 		return new ResponseEntity<>(responseBody, ex.getStatus().getHttpStatusCode());
 	}
 
+	@ExceptionHandler({Exception.class})
```

Comment:
```suggestion
        @ExceptionHandler({RestApiAuthException.class})
```

이렇게 해야 exception이 걸리는것 아닌가요.? 아닌가..?

Reply:
맞슴다맞슴다 복붙하다가 놓쳤네요 반영했습니다